pages/about/rdmkit_club.md

```diff
@@ -0,0 +1,20 @@
+---
+title: RDMkit Club
+---
+
+RDMkit Club is an informal drop-in session held on the **last Thursday of each month at 10:00 CEST/ 9:00 BST**, where we come together and discuss all topics related to RDMkit including its impact and future directions. 
```

Comment:
> at 10:00 CEST/ 9:00 BST**

Do we change these when the summertime is over? Use UTC?

Reply:
Fair point, if the online calculators are correct this will be 8h UTC, but does this solve the problem? Because summer or winter, it will be at 10h / 9h ?

Reply:
It is my understanding that UTC does not change depending on the time of the year, so that would solve the problem, I believe.

Reply:
The fact that it does not change, does that not create the problem? Sorry I feel stupid about this :D Because if CET is UTC +1 and CEST UTC +2, than it means with a fixed UTC hour, the hour of the meeting changes winter vs summer?

Reply:
The UTC can be very confusing!

Reply:
> The UTC can be very confusing!

Agreed, shall we use "CE(S)T" instead

Reply:
@bedroesb Thinking about this, I suspect you are right.

Reply:
I think the 10:00 CEST/ 9:00 BST option is more intuitive as well. Even if we forgot to change it, I think I'd find it easier to convert from BST to GMT than from UTC to whatever time zone we're in! It's what you're used to, I guess.

Reply:
@martin-nc What do you think of the CE(S)T solution, are is it abvious enough?

Reply:
I think so. Would you miss out BST/GMT then?